mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -32,10 +34,16 @@ CLASS net/minecraft/unmapped/C_jricjyva net/minecraft/block/Blocks
 		ARG 0 sticky
 	METHOD m_sidkwvli createStoneButton ()Lnet/minecraft/unmapped/C_mmxmpdoq;
 	METHOD m_tetaodtf rebuildShapeCache ()V
+	METHOD m_tldmwsps (Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_peaveboq;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_ogavsvbr;)Z
+		ARG 3 entityType
+	METHOD m_tobwsgju mapColorOf (Lnet/minecraft/unmapped/C_mlrprqox;)Ljava/util/function/Function;
```

Comment:
I think "create" is more conventional when not creating an instance of the class the method is in. #599 also mentions "waterlogged" for this.